src/hooks/useClearCookies.ts

```diff
@@ -0,0 +1,9 @@
+import Cookies from 'js-cookie';
+
+export function useClearCookies() {
+    // Removes cookies
+    Cookies.remove('secondsLeft');
+    Cookies.remove('gameCode');
+    Cookies.remove('isHost');
+
```

Comment:
Thanks for adding this! There are a few more cookies now from the 'client rejoin' PR. I'll add them in once this is merged